vck-rqes/src/commonMain/kotlin/at/asitplus/wallet/lib/Initializer.kt

```diff
@@ -0,0 +1,13 @@
+package at.asitplus.wallet.lib
+
+import at.asitplus.rqes.rdcJsonSerializer
+import at.asitplus.wallet.lib.data.serializerModuleCollection
+import kotlinx.serialization.modules.overwriteWith
+
+
+object Initializer {
+    fun initRqesModule() {
```

Comment:
So does this method need to be called by every client (i.e. applications including our library)? This needs to be documented, and included in the changelog.